client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -306,23 +311,25 @@ export class SeoForm extends React.Component {
 
 		const generalTabUrl = getGeneralTabUrl( slug );
 
-		const upsellProps = siteIsJetpack
-			? {
-					title: translate( 'Boost your search engine ranking' ),
-					feature: FEATURE_SEO_PREVIEW_TOOLS,
-					href: `/checkout/${ slug }/${ PRODUCT_UPSELLS_BY_FEATURE[ FEATURE_ADVANCED_SEO ] }`,
-			  }
-			: {
-					title: translate(
-						'Boost your search engine ranking with the powerful SEO tools in the Business plan'
-					),
-					feature: FEATURE_ADVANCED_SEO,
-					plan:
-						selectedSite.plan &&
-						findFirstSimilarPlanKey( selectedSite.plan.product_slug, {
-							type: TYPE_BUSINESS,
-						} ),
-			  };
+		const upsellProps =
+			siteIsJetpack && ! isAtomic
```

Comment:
Similarly to the analytics checks, it feels like this logic shouldn't be here. We should probably have the plan to promote, current plan, and site type all sent from the server with the other feature information, but that's definitely not something to tackle in this PR.

Maybe we should decide if that's a good idea and raise an issue.

Reply:
I opened an issue to track it: https://github.com/Automattic/wp-calypso/issues/53793